inbox/auth/generic.py

```diff
@@ -74,16 +77,17 @@ def authenticate_imap_connection(  # type: ignore[no-untyped-def, override]
         try:
             conn.login(account.imap_username, account.imap_password)
         except IMAPClient.Error as exc:
-            if auth_is_invalid(exc):
-                log.error(
-                    "IMAP login failed", account_id=account.id, error=exc
-                )
-                raise ValidationError(exc)  # noqa: B904
+            if is_error_message_invalid_auth(exc.args[0]):
```

Comment:
The original code in `auth_is_invalid` bailed out when `exc.args` was empty.
Should we do the same here?

Reply:
It can't be empty. It can contain an empty string, which will still return `False`.